abb_irb120_moveit_config/launch/trajectory_execution.launch.xml

```diff
@@ -7,9 +7,11 @@
   <param name="moveit_manage_controllers" value="$(arg moveit_manage_controllers)"/>
 
   <!-- When determining the expected duration of a trajectory, this multiplicative factor is applied to get the allowed duration of execution -->
-  <param name="allowed_execution_duration_scaling" value="1.2"/> <!-- default 1.2 -->
+  <param name="trajectory_execution/allowed_execution_duration_scaling" value="1.2"/> <!-- default 1.2 -->
   <!-- Allow more than the expected execution time before triggering a trajectory cancel (applied after scaling) -->
-  <param name="allowed_goal_duration_margin" value="0.5"/> <!-- default 0.5 -->
+  <param name="trajectory_execution/allowed_goal_duration_margin" value="0.5"/> <!-- default 0.5 -->
+  <!-- Disable execution duration monitoring to prevent trajectory cancel -->
+  <param name="trajectory_execution/execution_duration_monitoring" value="false" /> <!-- default false -->
```

Comment:
@shaun-edwards and @gavanderhoorn Is this the desired default behavior we want to have?